live-examples/js-examples/arraybuffer-constructor.html

```diff
@@ -0,0 +1,9 @@
+<pre>
+<code id="static-js">// create an ArrayBuffer with a size in bytes (in this case 8)
+var buffer = new ArrayBuffer(8);
+
+console.log(buffer);
+
+// expected output: [object ArrayBuffer]
+</code>
```

Comment:
I think this example is worth having. It shows us a valid argument to pass. Logging byteLength would be fine here too I think.